README.md

```diff
@@ -33,18 +33,19 @@ OpenSearch Continuous Integration is an open source CI system for OpenSearch and
 
 ### Dev Deployment 
 1. Setup your local machine to credentials to deploy to the AWS Account
-2. Deploy the ci-config-stackusing the with one of the following (takes ~1 minute to deploy) - 
+1. Deploy the bootstrap stack by running `cdk bootstrap` command that sets up required resources to create the stacks. [More info](https://docs.aws.amazon.com/cdk/latest/guide/bootstrapping.html) 
```

Comment:
We need to specify the params here as well, -c. Same goes for destroy as well. I have raised an issue to address this. #45

Reply:
bootstrap does not need a param. For other 2 stacks that can be addressed when someone picks up #45

Reply:
Lets switch all calls to only `npm run cdk...` variant so its consistant.  Advanced users that have cdk installed globally can do what they desire, but it makes it harder to read with effectively double the options

Reply:
I agree, hence the bug. Currently the check is to throw an exception irrespective of the cdk command. This needs to be tied to deploy only in the bug. Try running `cdk bootstrap` to confirm

Reply:
Added the documentation. Bug needs to address to throw the error only when required stack (CI-dev in this case) is deployed.